packages/shared/src/composables/use-items.ts

```diff
@@ -137,6 +138,7 @@ export function useItems(collection: Ref<string | null>, query: ComputedQuery, f
 				params: {
 					limit: unref(limit),
 					fields: fieldsToFetch,
+					...(alias && { alias: unref(alias) }),
```

Comment:
`alias` is extracted from the `query` parameter and is not required according to the `ComputedQuery` type so could be `undefined` im assuming.
```suggestion
...(alias ? { alias: unref(alias) } : {}),
```

Reply:
When it's `undefined`, it should short circuit due to the `alias &&` and not attempt to add the empty alias right? or perhaps this is meant as a different scenario from what I had in mind? Just wondering since this syntax had been used in a few places 🤔

Reply:
If it's `undefined` or any falsy value the expression will indeed short circuit to `false` but then try to unpack `...false` which would error.
![image](https://user-images.githubusercontent.com/9389634/172558810-9a7c1005-2a97-4c1f-b9d9-60271015eb75.png)

Reply:
Ahh finally get it! TIL